components/plan-build-ps1/bin/hab-plan-build.ps1

```diff
@@ -110,9 +110,11 @@ $script:pkg_expose = @()
 # An associative array representing configuration data which should be gossiped to peers.
 $script:pkg_exports = @{}
 # The user to run the service as
-$script:pkg_svc_user = "hab"
+$script:pkg_svc_user = ""
```

Comment:
In this case we want to set all `pkg_`* vars to ensure they ate `$script:` scoped By setting them in a `script` scope any use of them in plan_build will have script scope instead of scoped to individual functions.